src/permissions.rs

```diff
@@ -14,7 +15,7 @@ pub enum PermissionType {
 /// Decides if a GitHub user can perform various actions using the bot.
 #[async_trait]
 pub trait PermissionResolver {
-    async fn has_permission(&self, username: &str, permission: PermissionType) -> bool;
+    async fn has_permission(&self, _username: &UserId, _permission: PermissionType) -> bool;
```

Comment:
```suggestion
    async fn has_permission(&self, _user_id: &UserId, _permission: PermissionType) -> bool;
```
Same in a few other places.